idd/Energy+.idd.in

```diff
@@ -33627,15 +33627,18 @@ ZoneHVAC:UnitVentilator,
         \required-field
         \type node
    A8,  \field Outdoor Air Node Name
-        \required-field
         \type node
+        \note this field is left blank only if the Unit Ventilator is connected to a central
```

Comment:
Inlet Node Name field needs notes explaining that it must be a zone exhaust node if there is no DOA mixer or if there is an outlet side DOA mixer, but the unit ventilator air inlet node name must be the same as the mixer outlet node name for an inlet side mixer.  I/O ref description for the Inlet Node field needs the same notes.